src/SFA.DAS.Assessor.Functions/Functions/Print/PrintNotificationFunction.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.Domain.Print.Interfaces;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Print
+{
+    public class PrintNotificationFunction
+    {
+        private readonly IPrintNotificationCommand _command;
+
+        public PrintNotificationFunction(IPrintNotificationCommand command)
+        {
+            _command = command;
+        }
+
+        [FunctionName("PrintNotificationFunction")]
+        public async Task Run([TimerTrigger("0 */15 * * * *", RunOnStartup = true)]TimerInfo myTimer, ILogger log)
```

Comment:
we probably don't want to run this every 15 mins. I think once a day initially. Also we could put the schedule into configuration